app/models/flavor.rb

```diff
@@ -1,6 +1,8 @@
 class Flavor < ApplicationRecord
   include NewWithTypeStiMixin
   include CloudTenancyMixin
+  include AvailabilityMixin
```

Comment:
First comment in `AvailabilityMixin` says `# PLEASE PREFER supports_feature_mixin.rb OVER THIS`.

Is there reason why we should use `AvailabilityMixin` instead of recommended `SupportsFeatureMixin` from https://github.com/ManageIQ/manageiq/blob/master/app/models/mixins/supports_feature_mixin.rb ?

Maybe I am missing something...